docs/src/osdk/reference/commands/run.md

```diff
@@ -17,9 +17,10 @@ for more details.
 
 Options related with debugging:
 
-- `-G, --enable-gdb`: Enable QEMU GDB server for debugging.
-- `--vsc`: Generate a '.vscode/launch.json' for debugging kernel with Visual Studio Code
-(only works when QEMU GDB server is enabled, i.e., `--enable-gdb`).
+- `--gdb-server`: Enable QEMU GDB server for debugging.
+- `--gdb-wait-client`: Let the QEMU GDB server wait for the client connection before execution.
+- `--gdb-vsc`: Generate a '.vscode/launch.json' for debugging kernel with Visual Studio Code
+(only works when QEMU GDB server is enabled, i.e., `--gdb-server`).
```

Comment:
> only works when QEMU GDB server is enabled

Actually, this statement not only applies to `--gdb-vsc`, but also `--gdb-wait-client`, `--gdb-server-addr`. To make this more clear, I suggest refactoring the interface to the following form (that mimics the interface of QEMU).

```
--gdb-server [addr=ADDR][,support_vsc=on][,wait_client=on]
```

Without explaining, people will get it that the three options only apply to GDB server.